docs/data-sources/cassandra.md

```diff
@@ -68,14 +68,14 @@ Read-Only:
 
 Read-Only:
 
-- `additional_backup_regions` (List of String)
+- `additional_backup_regions` (Set of String)
 - `backup_hour` (Number)
 - `backup_minute` (Number)
 - `cassandra` (List of Object) (see [below for nested schema](#nestedobjatt--cassandra_user_config--cassandra))
 - `cassandra_version` (String)
-- `ip_filter` (List of String)
+- `ip_filter` (Set of String)
```

Comment:
changing `List` to `Set` would require to release a new major version since these types aren't backwards compatible with each other, and it could break manifests for some users that utilize some kind of the Collection Functions, e.g. [`sort`](https://developer.hashicorp.com/terraform/language/functions/sort), and some other ones

Reply:
Could you provide with a case? Worked for me.

Reply:
discussed internally that we will pass on it